pom.xml

```diff
@@ -95,6 +95,7 @@
         <jetty.version>9.2.10.v20150310</jetty.version>
         <pax.web.version>4.2.4</pax.web.version>
         <pax.cdi.version>0.12.0</pax.cdi.version>
+        <pax.logging.version>1.8.5</pax.logging.version>
```

Comment:
I had needed to use this version number in several places before when I was still modifying `startup.properties` and I didn't want to duplicate the version string everywhere. I can remove the property for the one reference left to the log4j2 bundle now.